ports/nrf/supervisor/port.c

```diff
@@ -295,6 +295,20 @@ void port_interrupt_after_ticks(uint32_t ticks) {
 }
 
 void port_sleep_until_interrupt(void) {
+#if defined(MICROPY_QSPI_CS) && defined(MICROPY_QSPI_OFF_WHEN_SLEEP)
```

Comment:
Please make this `qspi_disable` to match `qspi_enable` in `qspi_flash.c`.

Reply:
OK